cli/cloe_launch/exec.py

```diff
@@ -92,7 +93,7 @@ class Environment:
 
     def __init__(
         self,
-        env: Union[str, List[Path], Dict[str, str], None],
+        env: Union[Path, List[Path], Dict[str, str], None],
```

Comment:
Is Dict[str, str] correct? In line 496 we do cloe_env = Environment({}). Shoudn't it be just `env: Union[Path, List[Path], Dict, None],` ?

Reply:
Well:
- Since Python 3.9, we should just use `dict`. See: https://peps.python.org/pep-0585/
- We need to support the version of Python used in Ubuntu 18.04 at minimum, so: Python 3.8
- I see some uses in the internet with `Dict` by itself, but most with types: https://www.programcreek.com/python/example/94085/typing.Dict
- In the Python documentation, I don't get any clarification: https://docs.python.org/3/library/typing.html#typing.Dict

At any rate, an empty dict satisfies all type requirements. The type requirements of `Environment.__init__` say if there are elements in the dict, they should be `str` pointing to `str`.

Reply:
Understood. Thanks for the explanation!